tools/tf2openapi/generator/generate.go

```diff
@@ -2,8 +2,13 @@ package generator
 
 import (
 	pb "github.com/kubeflow/kfserving/tools/tf2openapi/generated/protobuf"
+	"github.com/kubeflow/kfserving/tools/tf2openapi/types"
 )
 
-func GenerateOpenAPI(model pb.SavedModel) string {
+func GenerateOpenAPI(model *pb.SavedModel) string {
+	m := types.NewTFSavedModel(model)
+	m.Schema()
```

Comment:
Does the Schema() call has any effect here? The return value of Schema() is not used anywhere in this method. (Might be some golang feature that I don't know of...)

Reply:
It doesn't have any effect; I call Schema() because "m" would be an unused variable otherwise, which would lead to a compilation error.

I could just return "", nil for this method until I have the OpenAPI generation code if that's cleaner. What do you think?